optimoveapi/api/actions.py

```diff
@@ -1,25 +1,37 @@
+import logging
+
 from ..utils import date_to_str
 
 
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.WARNING)
+
+
 class Actions(object):
     """Namespace class for action-related endpoints"""
 
     def __init__(self, transport):
         self._transport = transport
 
     def get_executed_campaign_details(self, date):
+        logger.debug('Getting executed campaign for {}'.format(date))
         path = 'actions/GetExecutedCampaignDetails'
         params = {
             'Date': date_to_str(date),
         }
-        return self._transport.get(path, params)
+        executed_campaign_details = self._transport.get(path, params)
+        logger.debug('Got {} executed campaign'.format(len(executed_campaign_details)))
```

Comment:
Following python logging cookbook, your logs fells rather into INFO type
DEBUG	Detailed information, typically of interest only when diagnosing problems.
INFO	Confirmation that things are working as expected.